player/low-latency-streaming/js/script.js

```diff
@@ -50,10 +63,7 @@ var conf = {
         muted: true,
     },
     adaptation: {
-        logic: 'low-latency-v1',
```

Comment:
why did you remove this. we need this.

Reply:
That was an accident caused by merging, nice catch will revert.